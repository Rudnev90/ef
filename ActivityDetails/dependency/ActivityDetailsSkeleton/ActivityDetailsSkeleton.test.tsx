
import * as React from 'react';

import { cleanup, render } from '@testing-library/react';
import renderer from 'react-test-renderer';
afterEach(cleanup);

import { ActivityDetailsSkeleton } from '.';

const ACTIVITY_DETAIL_SKELETON_BCS_ID = 'activity-details-skeleton';

describe('ActivityDetailsSkeleton', () => {
  it.concurrent('ActivityDetailsSkeleton render ok', async () => {
    const output = renderer.create(<ActivityDetailsSkeleton classes={{}} />);
    expect(output.toJSON()).toMatchSnapshot();
  });

  it.concurrent(`should have argument data-bcs-id="${ACTIVITY_DETAIL_SKELETON_BCS_ID}"`, async () => {
    const { getByTestId } = render(<ActivityDetailsSkeleton classes={{}} />);
    expect(getByTestId(ACTIVITY_DETAIL_SKELETON_BCS_ID)).toBeTruthy();
  });
});
