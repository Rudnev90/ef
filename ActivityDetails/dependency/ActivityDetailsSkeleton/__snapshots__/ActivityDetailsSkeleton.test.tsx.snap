// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div
  className="MuiGrid-container-1 MuiGrid-direction-xs-column-4"
  data-bcs-id="activity-details-skeleton"
>
  <div
    className="MuiGrid-item-2"
  >
    <div
      className="MuiGrid-container-1"
    >
      <div
        className="MuiGrid-item-2"
      >
        <div
          className="animated"
        />
      </div>
      <div
        className="MuiGrid-item-2"
      >
        <div
          className="MuiGrid-container-1 MuiGrid-direction-xs-column-4"
        >
          <div
            className="MuiGrid-item-2"
          >
            <div
              className="animated"
            />
          </div>
          <div
            className="MuiGrid-item-2"
          >
            <div
              className="animated"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-item-2"
  >
    <hr
      className="MuiDivider-root-98"
    />
  </div>
  <div
    className="MuiGrid-item-2"
  >
    <div
      className="MuiGrid-container-1"
    >
      <div
        className="MuiGrid-item-2"
      >
        <div
          className="animated"
        />
      </div>
      <div
        className="MuiGrid-item-2"
      >
        <div
          className="MuiGrid-container-1 MuiGrid-direction-xs-column-4"
        >
          <div
            className="MuiGrid-item-2"
          >
            <div
              className="animated"
            />
          </div>
          <div
            className="MuiGrid-item-2"
          >
            <div
              className="animated"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-item-2"
  >
    <hr
      className="MuiDivider-root-98"
    />
  </div>
  <div
    className="MuiGrid-item-2"
  >
    <div
      className="MuiGrid-container-1 MuiGrid-align-items-xs-center-9"
    >
      <div
        className="MuiGrid-item-2"
      >
        <div
          className="animated"
        />
      </div>
      <div
        className="MuiGrid-item-2"
      >
        <div
          className="animated"
        />
      </div>
    </div>
  </div>
</div>
`;
